packages/core/README.md

```diff
@@ -30,8 +30,9 @@ If you really need support for these legacy browsers, let us know, and we might
 
 ### Script tag
 
-- Put a script tag similar to this in the head of your index.html:
+- Just add the following tags to your page.
   ```html
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@w2wds/core@latest/dist/core/core.css" />
```

Comment:
I have to validate this when this PR is merged and I've done a publish to NPM. Doesn't work right now.